mysql-test/suite/sys_vars/r/sysvars_server_notembedded.result

```diff
@@ -2602,6 +2602,16 @@ NUMERIC_BLOCK_SIZE	1
 ENUM_VALUE_LIST	NULL
 READ_ONLY	NO
 COMMAND_LINE_ARGUMENT	REQUIRED
+VARIABLE_NAME	NEW_MODE
+VARIABLE_SCOPE	SESSION
+VARIABLE_TYPE	SET
+VARIABLE_COMMENT	Used to introduce new behavior to new MariaDB versions
+NUMERIC_MIN_VALUE	NULL
+NUMERIC_MAX_VALUE	NULL
+NUMERIC_BLOCK_SIZE	NULL
+ENUM_VALUE_LIST	NOW_DEFAULT,FIX_DISK_TMPTABLE_COSTS,FIX_SEMIJOIN_DUPS_WEEDOUT_CHECK,TEST_WARNING1,TEST_WARNING2
```

Comment:
Is it feasible to get rid of TEST_WARNINGs here?

Reply:
We don't want to see *NOW_DEFAULT* either, since it is pointless setting this.  Looking...